src/main.ts

```diff
@@ -27,11 +29,11 @@ import { Reference } from './reference';
  * given.
  */
 function parseSecretsRefs(secretsInput: string): Reference[] {
-  const secrets = new Array<Reference>();
-  for (const line of secretsInput.split(`\n`)) {
-    for (const piece of line.split(',')) {
-      secrets.push(new Reference(piece.trim()));
-    }
+  const list = parseCSV(secretsInput);
```

Comment:
Looks like we were previously splitting on newline but now using parseCSV

Reply:
I pulled this into the class and wrote unit tests for it.